graph-refresh/base/cronjob.yaml

```diff
@@ -46,11 +46,6 @@ spec:
                   value: "https://kubernetes.default.svc.cluster.local"
                 - name: KUBERNETES_VERIFY_TLS
                   value: "0"
-                - name: NAMESPACE
```

Comment:
why is the namespace env var removed?

Reply:
It was not required, we removed it from graph-refresh, see: https://github.com/thoth-station/graph-refresh-job/pull/545.

Thanks for review, I realized as I'm changing base, this will affect also prod env, should I create patches for cronjob.yaml only for test and stage? Or easier just to add a TODO to remove this unused env variable once prod will resemble stage and leave that env variable for now? 2 option also won't affect test and stage.

wdyt?

Reply:
I think we should go with option 2 as you suggested :)